integreat_cms/cms/urls/protected.py

```diff
@@ -139,6 +139,30 @@
         include(
             [
                 path("", settings.UserSettingsView.as_view(), name="user_settings"),
+                path(
+                    "totp/",
+                    include(
+                        [
+                            path(
+                                "",
+                                settings.TOTPRegisterView.as_view(),
+                                name="register_totp",
+                            ),
+                        ]
+                    ),
+                ),
+                path(
+                    "totp/",
+                    include(
+                        [
+                            path(
+                                "delete",
```

Comment:
```suggestion
                                "delete/",
```
I would prefer URLs ending with a trailing slash